pkg/commands/inspect.go

```diff
@@ -113,32 +112,30 @@ func (p *printer) zipDetails() error {
 	if err := p.title("Zip"); err != nil {
 		return err
 	}
-	p.line("ZIP_NAME=%s", p.build.Config().Parameters.ZipName)
-	return p.line("ZIP_PATH=%s", p.build.Config().Paths.ZipPath)
+	if err := p.line("ZIP_NAME=%s", p.build.Config().Parameters.ZipName); err != nil {
+		return err
+	}
+	if err := p.line("ZIP_PATH=%s", p.build.Config().Paths.ZipPath); err != nil {
+		return err
+	}
+	return nil
 }
 
-func firstErr(f ...func() error) error {
-	for _, ef := range f {
-		if err := ef(); err != nil {
+func (p *printer) worktreeStatus() error {
+	if err := p.title("Worktree Status"); err != nil {
+		return err
+	}
+	product := p.build.Config().Product
+	if !product.IsDirty() {
+		return p.line("clean")
+	}
+	if err := p.line("dirty files:"); err != nil {
+		return err
+	}
+	for _, f := range product.DirtyFiles {
+		if err := p.line(f); err != nil {
 			return err
 		}
 	}
 	return nil
 }
-
-func (p *printer) title(s string) error {
-	if !p.printTitles {
-		return nil
-	}
-	_, err := fmt.Fprintln(p.w, s+":")
-	return err
-}
-
-func (p *printer) line(s string, a ...any) error {
-	_, err := fmt.Fprintf(p.w, p.prefix+s+"\n", a...)
-	return err
-}
-
-func tabWrite[T any](p *printer, list []T, line func(T) string) error {
-	return cli.TabWrite(p.w, list, line)
-}
```

Comment:
Deleted code here was moved to `printer.go` since it's generic and could be used by other commands.